Jenkinsfile

```diff
@@ -140,11 +142,15 @@ spec:
     imagePullPolicy: Always
     env:
       - name: JAVA_TOOL_OPTIONS
-        value: -Xmx6G
+        value: -Xmx1G
     resources:
       limits:
-        memory: "10Gi"
-        cpu: "2.0"
+        memory: "6Gi"
+        cpu: "1.0"
+      requests:
+        cpu: "0.5"
```

Comment:
Here we are saying that each TCK test will be allocated 1/2 of a CPU and initially 4gb of memory (we can grow to use 1 full CPU and 6gb of memory if there are enough resources).